backend/src/render/csv.mjs

```diff
@@ -0,0 +1,42 @@
+import { createArrayCsvStringifier } from 'csv-writer';
+import { BillingMode } from '@aws-sdk/client-dynamodb';
+import {scyllaPricingUrl} from "./common.mjs";
+
+/**
+ * @param {{ onDemand: Usages, provisioned: Usages }} tableCosts
+ */
+export const renderCsv = ({ onDemand, provisioned }) => {
+  const csv = createArrayCsvStringifier({
+    header: ['Table name', 'Billing mode', 'Table size', 'Average item size', 'RCU', 'WCU', 'DynamoDB comparison URL']
+  });
+  const allRecords =
+    usageRecords(provisioned, BillingMode.PROVISIONED)
+      .concat(usageRecords(onDemand, BillingMode.PAY_PER_REQUEST));
+  return csv.stringifyRecords(allRecords);
+};
+
+/**
+ * @param {Usages} usages
+ * @param {BillingMode} billingMode
+ * @return {Array<[string, string, string, string, string, string, string]>}
+ */
+const usageRecords = (usages, billingMode) =>
+  usages.tableUsages.map(table => {
+    return [
+      table.name,
+      billingMode,
+      table.sizeBytes,
+      Math.round(table.averageItemSizeBytes),
+      Math.round(table.rcu),
+      Math.round(table.wcu),
+      ''
+    ]
+  }).concat([[
+    'TOTAL',
+    billingMode,
+    usages.summary.sizeBytes,
+    Math.round(usages.summary.averageItemSizeBytes),
+    Math.round(usages.summary.readThroughputBytes / 4096),
+    Math.round(usages.summary.writeThroughputBytes / 1024),
+    scyllaPricingUrl(usages.summary)
```

Comment:
Yep, this is nice. IIUC here we aggregate everything under a single url, right? Point back to my other comment about different links for on-demand / provisioned on HTML output.

Reply:
We use two URLs in total: one for on-demand tables, and one for provisioned tables. So, this is the same as with the HTML report. I agree with your concerns expressed above.